src/templates/_macros/svg_macros.html

```
@@ -0,0 +1,137 @@
+                        <th>{{ lang }}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for row in translations %}
+                    <tr>
+                        <td> {{ loop.index }} </td>
+                        <td>
+                            {% set en_text = row.get('en', '') %}
+                            <small class="text-muted">
+                                {# {{ en_text[:80] }}{% if en_text|length > 80 %}...{% endif %} #}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)